docfx_yaml/extension.py

```diff
@@ -498,7 +555,8 @@ def _extract_docstring_info(summary_info, summary):
 
     # Add extracted summary
     if lines != []:
-        lines = _resolve_reference_in_module_summary(lines)
+        for PATTERN in [REF_PATTERN, REF_PATTERN_LAST]:
+            lines = _resolve_reference_in_module_summary(PATTERN, lines)
```

Comment:
Drop the for loop and call the function directly twice? I was confused at first with what was happening here.

Reply:
done. I'll add comments as well.